### PR TITLE
Updating getting started guide with more mention of custom.init. 

### LINKED ISSUE #241: Update custom.init docs to mention new --dir flag
https://luckyframework.org/guides/getting-started/starting-project

We now have a custom `--dir` flag on the custom.init

### PR DESCRIPTION
**Pull request by jwoertink**
Fixes #241

The custom.init was just a side note before, but now it has it's own section with a little more detail as well as mentioning the new `--dir` flag option.

### CODE REVIEW

File: src/actions/guides/getting-started/starting_project.cr [hunk 1]
```diff
@@ -28,8 +28,26 @@ class Guides::GettingStarted::StartingProject < GuideAction
...
-    > pass `--api` or `--no-auth` as options. Use the `-h` flag for more help.
+    ## Skipping the Wizard
+
+    If you'd like to skip the wizard, you can run `lucky custom.init my_app`. This option supports
+    all of the same options as the wizard does, but also with some additional flags for quick customization.
+
+    This option is great if you need to generate a Lucky app programmatically, or just prefer to get going right away.
+
+    ```
+    # Generate an API only app
```
Comment (paulcsmith) [inline src/actions/guides/getting-started/starting_project.cr:42]: I think showing with just project name here would be great. A lot of people skip to the code section

**Review by paulcsmith — APPROVED** [review]
Just one comment. Looks good to go when you’re ready!

File: src/actions/guides/getting-started/starting_project.cr (hunk 1 above)
Comment (jwoertink) [reply src/actions/guides/getting-started/starting_project.cr:42]: You mean an example of just `lucky custom.init my_app`?

Comment (paulcsmith) [reply src/actions/guides/getting-started/starting_project.cr:42]: Yep! I know you have it above but having it here with the others would be helpful I think. A lot of people skip straight to the high contrast code sample

**Review by paulcsmith — APPROVED** [review]
Love it! LGTM

**Pull request merged**